tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -2168,6 +2185,15 @@ Status FragmentMetadata::write_generic_tile_offsets(Buffer* buff) const {
   return Status::Ok();
 }
 
+Status FragmentMetadata::write_array_schema_name(Buffer* buff) const {
+  uint64_t size = array_schema_name_.size();
+  Status status = buff->write(&size, sizeof(uint64_t));
```

Comment:
Use `RETURN_NOT_OK` macros for cases like this (please also check elsewhere in the code diff).

Reply:
Will correct it and other cases